stories/features/newTab/default/index.tsx

```diff
@@ -5,7 +5,7 @@
 import * as React from 'react'
 
 // Feature-specific components
-import Clock from '../../../../src/old/clock'
+import Clock from '../../../../src/features/newTab/default/clock/index'
```

Comment:
please change it to `import { Clock } from '../../../../src/features/newTab/default/clock'`